aikau/src/main/resources/alfresco/layout/AlfSideBarContainer.js

```diff
@@ -128,6 +155,16 @@ define(["dojo/_base/declare",
        * @default
        */
       initialSidebarWidth: 350,
+
+      /**
+       * Indicates whether or not the sidebar should be resizable or not.
+       * 
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      isResizeable: true,
```

Comment:
The rest of Aikau uses "resizable" without the extra "e", as in the comment for this variable, so I think we should be consistent here and drop the extra "e".